eng/pipelines/upload-build-artifacts-jobs.yml

```diff
@@ -0,0 +1,167 @@
+parameters:
+  runtimeRepoAlias: runtime
+  performanceRepoAlias: self
+  buildConfiguration: []
+  mauiFramework: ''
+
+jobs:
+  - ${{ if containsValue(parameters.buildConfiguration, 'coreclr_arm64_linux') }}:
+    - template: /eng/pipelines/templates/upload-build-artifacts-job.yml@${{ parameters.performanceRepoAlias }}
+      parameters:
+        buildConfiguration: 'coreclr_arm64_linux'
+        dependencyJobName: build_linux_arm64_release_coreclr
+        artifacts:
```

Comment:
Is there any way that we can build these strings from the configuration, and not rely on hard coding the value?